src/app/about/about.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/personalmoney/app-supabase/pullRequest?prid=7587366)
```suggestion
import { Component, OnInit } from "@angular/core";
```